interfaces/builtin/opengl.go

```diff
@@ -183,6 +183,48 @@ unix (bind,listen) type=seqpacket addr="@cuda-uvmfd-[0-9a-f]*",
 # From https://bugs.launchpad.net/snapd/+bug/1862832
 /run/nvidia-xdriver-* rw,
 unix (send, receive) type=dgram peer=(addr="@var/run/nvidia-xdriver-*"),
+
+# Hybris support
+/{,var/}run/shm/hybris_shm_data rw, # FIXME: LP: #1226569 (make app-specific)
+# This is a LD_PRELOADed TLS padding library allowing use of both
+# Android 9 and glibc TLS slots at the same time. Only used on Halium 9.
+/var/lib/snapd/hostfs/usr/lib/@{multiarch}/libtls-padding.so  mr,
```

Comment:
Again this should be removed since it appears unnecessary

Reply:
Please check and address my previous comment above, we still need a solution for Mediatek Android 10 devices and I doubt waiting for much longer is a good idea.

Reply:
Alright, short-term I am fine without supporting MTK-10 devices for now, but I would really like to have this handled later.

Handled in d5a85d6